linkml/generators/jsonschemagen.py

```diff
@@ -39,6 +39,9 @@
     "xsddate": ("string", "date"),
     "xsddatetime": ("string", "date-time"),
     "xsdtime": ("string", "time"),
+    "uri": ("string", "uri"),
+    "uriorcurie": ("string", "uri"),
```

Comment:
based on the latest comments in the issue, I think this line and the succeeding one should be removed?

Reply:
True, I realized that later.
But since I also identified a deeper rooted issue (that I am trying to address in [this PR](https://github.com/linkml/linkml-model/pull/202/commits/bd5526536b0509e9bc26568ab5c0a40dd24375d0)) that has an impact on this PR, I left this PR unfixed.